### ISSUE TITLE
Vdiskcontroller extend does not display on GUI

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#924]
### Steps

Found during verification of https://github.com/openvstorage/framework/issues/892
- Installed OVH
- Setup disk roles
- Setup backend
- Setup vpool
- Added vdisk
- Used postman to send api call
  ![selection_081](https://cloud.githubusercontent.com/assets/17570109/18630891/49780fae-7e6f-11e6-9bd6-9f74c75fe414.png)
### Output

Tested two cases: a smaller volume and a bigger volume

**Smaller volume:** 
Fetching task details:

```
{
  "status": "FAILURE",
  "successful": false,
  "failed": true,
  "result": "Shrinking is not possible",
  "ready": true,
  "id": "45372824-7749-457f-93bb-673b729e3aa0"
}
```

**Bigger volume:** 
Fetching task details:

```
{
  "status": "SUCCESS",
  "successful": true,
  "failed": false,
  "result": null,
  "ready": true,
  "id": "45dfe5b3-57e5-498e-84bd-8127492a584e"
}
```

Found the following in volume driver logging:

```
2016-09-19 13:42:18 888886 +0200 - ovs-node1 - 1604/0x00007f397effd700 - volumedriverfs/XMLRPCTimingWrapper - 0000000000002766 - info - execute: Arguments for resizeObjec
t are {[volume_id:d0445b8a-7a30-49b2-b57e-bf803c436f41,volume_size:3221225472B,vrouter_cluster_id:94d36c61-1951-44af-8fe6-8c2441b8d8ea]}
2016-09-19 13:42:18 891044 +0200 - ovs-node1 - 1604/0x00007f397effd700 - volumedriverfs/XMLRPCTimingWrapper - 0000000000002767 - info - execute: Call resizeObject took 0.
002092 seconds
```

Task is correctly processed.

**Verification on GUI:**:
- The old volume size was 2GB
- New size _should_ be 3GB

Found:
![selection_082](https://cloud.githubusercontent.com/assets/17570109/18631026/fd954e48-7e6f-11e6-8e58-fd44a5118207.png)

```
root@ovs-node2:~# ls -al /mnt/vm-vpool/test1.raw 
-rwxrwxr-x 1 ovs ovs 3221225472 Sep 19 11:57 /mnt/vm-vpool/test1.raw
```

**Test succeeded for this scope: however expected 3GB on GUI and got 2GB**
### Test result

Test succeed. However please note that the size was not displayed correctly on the GUI.

Hyperconverged setup
- Three nodes with each three disks for the back-end
### Package information
- openvstorage                       2.7.3-rev.3978.4f93e02-1                                 amd64        openvStorage
- openvstorage-backend               1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.3-rev.3978.4f93e02-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-health-check          2.0.0-rev.134.9ef41c0-1                                  amd64        Open vStorage HealthCheck
- openvstorage-sdm                   1.6.3-rev.357.ee7c9f5-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-test                  2.7.3-rev.1012.608f550-1                                 amd64        openvStorage autotest suite
- openvstorage-webapps               2.7.3-rev.3978.4f93e02-1                                 amd64        openvStorage Web Applications

**Comment by khenderick** [comment]
Probably the volumedriver doesn't send the resize event when a voldrv client truncate is executed.

**Comment by saelbrec** [comment]
To my knowledge the events are only triggered when passing through the filesystem layer. Why doesn't the Framework vDiskController update the size in the model upon successful completion of the storageDriverClient.truncate call ?

**Comment by khenderick** [comment]
That's indeed exactly what needs to happen. Just too much assumptions during the move to using the volumedriver API.

**Comment by khenderick** [comment]
Fixed by #926, packaged in openvstorage-2.7.3-rev.3988.2d9eb6a

**Comment by JeffreyDevloo** [comment]
I like to point out that when I upgraded my ovs, the volume was **NOT** changed on the GUI. I had to resize it once more to get the value to update.

### Steps
- Upgraded the environment where the issue took place
- Sent a similar postman package
  ![selection_088](https://cloud.githubusercontent.com/assets/17570109/18637615/446ea414-7e8d-11e6-804e-6829279d8623.png)

### Output

![selection_089](https://cloud.githubusercontent.com/assets/17570109/18637622/4a8660bc-7e8d-11e6-844f-f8a7fbf4bdba.png)

### Verification

**Expecting 4GB on GUI**
![selection_090](https://cloud.githubusercontent.com/assets/17570109/18637638/5e7176e8-7e8d-11e6-8718-414327b523e3.png)

Verifying it on the OS aswell

```
root@ovs-node1:~# ls -alh /mnt/vm-vpool/test1.raw 
-rwxrwxr-x 1 ovs ovs 4.0G Sep 19 11:57 /mnt/vm-vpool/test1.raw
```

### Test result

Test passed.

### Setup

Hyperconverged setup
- Three nodes with each three disks for the back-end

### Package information
- openvstorage                       2.7.3-rev.3988.2d9eb6a-1                                 amd64        openvStorage
- openvstorage-backend               1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.3-rev.3988.2d9eb6a-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.3-rev.709.5bb94c8-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-health-check          2.0.0-rev.134.9ef41c0-1                                  amd64        Open vStorage HealthCheck
- openvstorage-sdm                   1.6.3-rev.357.ee7c9f5-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-test                  2.7.3-rev.1012.608f550-1                                 amd64        openvStorage autotest suite
- openvstorage-webapps               2.7.3-rev.3988.2d9eb6a-1                                 amd64        openvStorage Web Applications
